spec/acceptance/email_notification_spec.rb

```diff
@@ -1,6 +1,4 @@
 RSpec.describe App do
-  before { ENV['AUTHORISED_EMAIL_DOMAINS_REGEX'] = '.gov.uk$' }
```

Comment:
This did nothing

Reply:
Is that because this REGEX env-var is set in a Dockerfile somewhere? I don't like specs relying on Dockerfiles, makes the setup distant from the assertion.